src/core/gui/sidebar/previews/base/SidebarPreviewBaseEntry.cpp

```diff
@@ -20,7 +20,7 @@ SidebarPreviewBaseEntry::SidebarPreviewBaseEntry(SidebarPreviewBase* sidebar, co
     this->widget = gtk_button_new();  // re: issue 1072
 
     gtk_widget_show(this->widget);
-    g_object_ref(this->widget);
+    g_object_ref_sink(this->widget);
```

Comment:
**Nitpick**: 
The reference can be sunk and assigned in one line:

`this->widget = g_object_ref_sink(gtk_button_new());`